js/flightlog.js

```diff
@@ -1147,7 +1147,16 @@ FlightLog.prototype.rcCommandRawToThrottle = function(value) {
 
 FlightLog.prototype.rcMotorRawToPct = function(value) {
     // Motor displayed as percentage
-    return Math.min(Math.max(((value - this.getSysConfig().motorOutput[0]) / (this.getSysConfig().motorOutput[1] - this.getSysConfig().motorOutput[0])) * 100.0, 0.0),100.0);
+    let motorPct;
+    if (this.getSysConfig().fast_pwm_protocol < FAST_PROTOCOL.indexOf('DSHOT150')) {
+        // Analog protocol
+        const ANALOG_RANGE = this.getSysConfig().maxthrottle - ANALOG_MIN_VALUE;
+        motorPct = ((value - ANALOG_MIN_VALUE) / ANALOG_RANGE) * 100;
+    } else {
+        // Digital protocol
+        motorPct = ((value - DSHOT_MIN_VALUE) / DSHOT_RANGE) * 100;
+    }
+    return Math.min(Math.max(motorPct, 0.0), 100.0);
```

Comment:
With a float we get more precision

PWM | Dshot | MotorPct
-- | -- | --
1000 | 0 | -2.40120
1001 | 48 | 0.00000
1002 | 50 | 0.10010
1003 | 52 | 0.20020
1004 | 54 | 0.30030
1005 | 56 | 0.40040
1006 | 58 | 0.50050
1007 | 60 | 0.60060
1008 | 62 | 0.70070
1009 | 64 | 0.80080
1010 | 66 | 0.90090
1011 | 68 | 1.00100
1012 | 70 | 1.10110
1013 | 72 | 1.20120
1014 | 74 | 1.30130
1015 | 76 | 1.40140